### ISSUE TITLE
[FEATURE] Alliance Diplomacy view should show war declarations

### ISSUE THREAD

**Issue by mys** [steemnova/steemnova#14]
Only 'alliance' and 'non-agression' agreements are listed now.

**Comment by etaletai13** [comment]
Is the data easily pulled? I'm all about that kind of job, like kerning.

**Comment by mys** [comment]
Yes, the agreements between alliances are stored in SQL table "_diplo". "Level" number is a agreement type, eg. war or non-agreement pact.

**Comment by etaletai13** [comment]
Fantastic, I'll fritter away on this one and hope somebody else does a better job of it as well. It'd be nice to see a couple of the feature additions coming from different vantage points.
